src/mission_companion.cpp

```diff
@@ -1717,7 +1721,13 @@ void talk_function::field_harvest( npc &p, const std::string &place )
             tmp.charges = 1;
         }
         player_character.i_add_or_drop( tmp, number_seeds );
-        add_msg( _( "You receive %d %s…" ), number_seeds, tmp.type_name( 3 ) );
+        add_msg( _( "You receive %d %s…" ), number_seeds, tmp.type_name( number_seeds ) );
```

Comment:
I tried adding this same phrasing to `iexamine::harvest_plant` but realized it sounds bad with charged items (12 pumpkin is a single pumpkin) and items where a single item/charge is plural (e.g. seeds) and also sometimes the "handful of item" item names.
A few of us brainstorming on Discord couldn't come up with a good solution that covers all the bases. I don't like leaving these two cases different, so I'm going to think further on it and standardize them on... something.